src/docs/product/sentry-basics/grouping-and-fingerprints/index.mdx

```diff
@@ -11,21 +11,21 @@ description: "Learn more about how Sentry groups issues together as well as diff
 Have you ever had a set of similar-looking issues like this?
 ![Issues Dashboard](issues-dashboard.png)
 
-Let's first understand how events group into an issue. The issue `testTypeIssue16` has 18 events that Sentry grouped into an issue because they share the same _fingerprint_.
+Let's first understand how errors and transactions group into an issue. For example, the issue `testTypeIssue16` has 18 events that Sentry grouped into an issue because they share the same _fingerprint_.
 
 **What is a fingerprint?**
 A fingerprint is a way to uniquely identify an event. Sentry sets a fingerprint by default for you. Sentry uses its own built-in grouping algorithms to generate a fingerprint based on information available within the event such as a stack trace, exception, and message. Events with the same fingerprint are grouped together.
 
-**Do I need to do anything?** No. A stack trace is used by default if it's available. So if two events have the same stack trace, then they will group into one issue. If you have separate issues that you'd like to group together, then this guide will show you how.
+**Do I need to do anything?** No. Sentry automatically generates a fingerprint based on the event type and all available information. If events have the same root cause, they will group into one issue. If you have separate issues that you'd like to group together, then this guide will show you how.
 
-**How do I see the fingerprint?** Open an issue, click the JSON link, and find the _fingerprint_ property. If the default grouping was used, you'll see default written there. If a different grouping was used, you'll see the actual fingerprint value itself.
+**How do I see the fingerprint?** Open an issue, scroll down to the "Event Grouping Information" section, and click "Show Details". You'll see detailed information about how the fingerprint was generated, the algorithm that was used, and the final result.
```

Comment:
We added this bit based on feedback about it being challenging to find this information, so I don't want to lose it entirely, though I see why you need to make this edit. What about adding it [here](https://docs.sentry.io/product/data-management-settings/event-grouping/fingerprint-rules/) after the first paragraph? Something like, "to see the current fingerprint..."

Reply:
Ah, gotcha! I reverted this change for now. The "Fingerprinting Rules" page is about to get a "This is For Errors Only" bar at the top, so putting information about the "Event Grouping Information" section might be weird. How about adding it to the "Issue Details Page" and "Performance Issue Details Page" pages, since that's where this UI lives?

Reply:
Done!